experiment/measurer/measure_manager.py

```diff
@@ -574,9 +615,9 @@ def get_fuzzer_stats(stats_filestore_path):
     return json.loads(stats_str)
 
 
-def measure_trial_coverage(  # pylint: disable=invalid-name
-        measure_req, max_cycle: int,
-        q: multiprocessing.Queue) -> models.Snapshot:
+def measure_trial_coverage(  # pylint: disable=invalid-name,too-many-arguments
+        measure_req, max_cycle: int, q: multiprocessing.Queue,
+        process_specific_df_containers) -> models.Snapshot:
```

Comment:
Don't really know what "process" means here.

Reply:
Will changed it to "trial". "process" here was to indicate that it's specific to a single thread.